src/api/packers/FixedOrderedPacker.ts

```
@@ -0,0 +1,445 @@
+					// fits more blocks in the same or less size
+					blocks = new_blocks;
+				}
+			}
+			if(blocks == null) {
+				throw new Error("No blocks found");
+			}
+		}
+
+		if(blocks == null) {
+			throw new Error("No blocks found");
+			return [];
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)